optimum/exporters/neuron/__main__.py

```diff
@@ -475,6 +488,7 @@ def load_models_and_neuron_configs(
     lora_weight_names: Optional[Union[str, List[str]]],
     lora_adapter_names: Optional[Union[str, List[str]]],
     lora_scales: Optional[Union[float, List[float]]],
+    tensor_parallel_size: int = 1,
```

Comment:
I thinks this parameter is Optional, since you provide a default value. There a re multiple instances of this.

Reply:
It's literally optional but not `Optional` since Optional means that it could be `None`, but it couldn't in my implementation.